domain/src/main/java/com/nextstep/edu/domain/Expression.kt

```diff
@@ -0,0 +1,60 @@
+package com.nextstep.edu.domain
+
+import java.util.Optional
+
+class Expression(
+    operands: String = ""
```

Comment:
```suggestion
    expressions: String = ""
```

의미에 맞게 같이 수정되면 좋을 것 같습니다~

Reply:
operands도 expressions로 수정하는게 의미에 맞겠네요!! 감사합니다~